src/helpers/ConfigHelper.ts

```diff
@@ -33,13 +40,186 @@ export interface IntegrationCapabilities {
 const MAX_CATEGORIES = 10;
 
 export class ConfigHelper {
+  /**
+   * convertConfigToVersionOne - converts v0 schema to v1 schema format
+   *
+   * v0 schema example
+   * ---
+   *  promptOptions: {
+   *      acceptButtonText: '',
+   *      cancelButtonText: '',
+   *      actionMessage   : '',
+   *      slidedown: {...}
+   *  }
+   *
+   * v1 schema example
+   * ---
+   * "promptOptions": {
+   *    "slidedown": {
+   *      "enabled": true,
+   *      "autoPrompt": true,
+   *      "acceptButtonText": "",
+   *      "cancelButtonText": "",
+   *      "actionMessage": "",
+   *      "...",
+   *    }
+   * }
+   * @param  {any} promptOptions
+   * @returns AppUserConfigPromptOptions
+   */
+  private static convertConfigToVersionOne(promptOptions: any) : AppUserConfigPromptOptions {
+    if (!this.isPromptOptionsVersion0(promptOptions)) {
```

Comment:
We can remove this check, we already call `isPromptOptionsVersion0` at all entry points.